src/components/molecules/NHSNotifyAuthenticator/NHSNotifyAuthenticator.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable @typescript-eslint/no-require-imports,unicorn/prefer-module */
```

Comment:
Are these needed? 

If they are and you think they're silly rules, then disable them globally?

Reply:
these disables are not needed. in general, i think these are good rules, it's just unfortunate that there isn't a better way than require() to see the amplify outputs file